### ISSUE TITLE
An error occurred (ValidationError) when calling the CreateStack operation: Template format error: Every AllowedValues member must be a list.

### ISSUE THREAD

**Issue by dragonfly-net** [aws-cloudformation/cfn-python-lint#1295]
cfn-lint 0.26.2

Checking .yaml file, block 
```yaml
Parameters:
  Env:
    Type: "String"
    Description: "Environment"
    Default: "staging"
    AllowedValues:
            staging
            mock
            production
            DEVOPS
```
Lint says no errors, but when run aws cloudformation:
An error occurred (ValidationError) when calling the CreateStack operation: Template format error: Every AllowedValues member must be a list.

**Comment by kddejong** [comment]
Should be an easy one to do.  Thanks for submitting this.

**Comment by kddejong** [comment]
after this PR is in.

```
E2001 Property Parameters/Env/AllowedValues should be of type List
test/fixtures/templates/bad/parameters/configuration.yaml:Line:Column
```

**Comment by arbermejo0417** [comment]
Hi I'm having the same error when trying to deploy a stack

```
Parameters:
  EnvironmentSize:
    Type: String
    Default: SMALL
    AllowedValues:
      - SMALL
      - MEDIUM
      - LARGE
```
